src/index.js

```diff
@@ -221,6 +221,36 @@ const mini = (tree, {mini}) => {
   return bypass(tree);
 };
 
+const sortAttr = (tree, {rules: {sortAttr}}) => {
+  tree.map(node => {
+    if (node.attrs && Object.keys(node.attrs).length > 1 && sortAttr) {
```

Comment:
I think it's better to check in a slightly different sequence
1. `sortAttr` If this value is false there is no need to check for step two
2. `node.attrs`
3. `Object.keys(node.attrs)` It is better to put it in a separate constant so that you don't create an array of keys twice

Reply:
ahh, agreed 👍 .

I will change it 👍